apps/myaccount/src/api/validation.ts

```diff
@@ -147,15 +147,28 @@ export const getUsernameConfiguration = (configs: ValidationDataInterface[]): Va
     const usernameConf: ValidationDataInterface[] =
         configs?.filter((data: ValidationDataInterface) => data.field === "username");
 
-    if (usernameConf === undefined || usernameConf.length < 1) {
-        return;
+    if (usernameConf === undefined || usernameConf.length < 1
+        || usernameConf[0].rules === undefined || usernameConf[0].rules.length < 1) {
```

Comment:
We can shorten this condition like this, can't we?

```suggestion
    if ( !usernameConf?.length || !usernameConf[0]?.rules.length ) {
```